website/content/en/docs/ansible/development-tips.md

```diff
@@ -202,6 +202,9 @@ annotations:
   ansible.operator-sdk/reconcile-period: "30s"
 ```
 
+Note that a lower period will correct entropy more quickly, but reduce responsiveness to change 
```

Comment:
Lets remove this note, and document this thoroughly in 1 place, the reference.

Reply:
I think it's ok to leave it here for now, when we reorg we can consolidate everything well but I think users have a better chance of seeing it right here for now